roles/collect_modified_images/tasks/parse_image_names.yml

```diff
@@ -0,0 +1,9 @@
+- name: "split item"
+  set_fact:
+    timestamp: "{{ item.split(',')[0] }}"
+    image: "{{ item.split(',')[1][15:-1] }}"
+
+- name: "parse and return image name when timestamp is after start_time"
+  set_fact:
+    IMAGES: "{{ IMAGES + [image] }}"
+  when: "{{ timestamp > deployment_start_time }}"
```

Comment:
@14rcole Does the timestamp match with ansible .iso8601 standard ?? 
Which timezone will be taken in both cases ?
is nt it good for us to use utc times ?

Reply:
Both strings are in iso8601, but the timezones are off, good catch.  It looks like openshift uses UTC, but ansible uses the local timezone.  Do you know which version of ansible CVP uses?  There's an easy fix, but it requires 2.8 or higher.

Reply:
@14rcole We are using 2.7 in CVP right now. 
However, In order to get the UTC time i think i have a workaround without moving to 2.8 . 

```yaml
 - name: "Get shell date utc"
      shell: "date -u '+%Y-%m-%d %H:%M:%S'"
      register: date_via_shell
```
You can always get the utc time using shell date command.